packages/http-client-java/emitter/src/type-utils.ts

```diff
@@ -217,8 +218,9 @@ export function getUnionDescription(union: Union, typeNameOptions: TypeNameOptio
   return name;
 }
 
-export function modelIs(model: Model, name: string, namespace: string): boolean {
-  let currentModel: Model | undefined = model;
+export function modelIs(model: SdkModelType | undefined, name: string, namespace: string): boolean {
+  // use raw model because SdkModelType does not have sourceModel information
```

Comment:
Currently I'm using raw model because  `SdkModelType` does not have `sourceModel` information and this is by TCGC's design. It could be better to use `model.crossLanguageDefinitionId` to do the comparison. But for `model A is Azure.Core.Foundations.OperationStatus` case, the model A will be generated.

Reply:
Yeah, this handle the sourceModel too.

Wondering whether TCGC have similar function.

Reply:
No, TCGC does not have similar function. They said for `model A is Azure.Core.Foundations.OperationStatus` case, model `A` be generated is expected. If we really want to no generate model `A`, we need to use raw. /cc @tadelesh